install/zowe-install.yaml.template

```diff
@@ -7,6 +7,8 @@ install:
   userDir=~/zowe-user-dir
   prefix=ZOWE
   instance=1
+  # Dataset prefix of SZWE* data sets:  
+  datasetPrefix={hlq}.ZWE
```

Comment:
The default should be `datasetPrefix={userid}.ZWE` and the script pick up that it's userid and determine the current user,  or else if the value is changed then use the changed value instead as an explicit HLQ.ZWE

Reply:
I ran build 8 and got the following error:
```Creating MVS artefacts ... 
  SZWEAUTH not created, RC=12
  SZWESAMP not created, RC=12```
It logged ZOWE_DSN_PREFIX={userid}.ZWE, so it looks like username replacement isn't working

Reply:
Line #10 of the template in branch `PDS-LOADLIB-SAMPLIB` is datasetPrefix={userid}.ZWE, and line #374 of install-zowe.sh in branch `add-HLQ-DSN` performs the replacement.

Reply:
@John-A-Davies - install-zowe.sh is a test script, so completely irrelevant to the fact that the code does not work for me.

Reply:
DSN-prefix is working, but I'm still getting errors in the dataset creation:
```
Creating MVS artefacts ... 
  SZWEAUTH created
  ZWESIS01 copied to SZWEAUTH
  SZWESAMP created
cp: /u/stevenh/749/zowe-1.5.0/files/templates/ZWESECUR.jcl: EDC5129I No such file or directory.
  ZWESIP00 copied to SZWESAMP
  ZWESISCH copied to SZWESAMP
  ZWESISMS copied to SZWESAMP
  ZOWESVR copied to SZWESAMP
cp: /u/stevenh/749/zowe-1.5.0/temp_2019-10-03/files/zss/SAMPLIB/ZWESECUR: EDC5129I No such file or directory.
  ZWESECUR not copied to SZWESAMP, RC=1
  ZWESISTC copied to SZWESAMP
  ZWESIPRG copied to SZWESAMP
```

Reply:
The automated builds have the same problem: https://wash.zowe.org:8443/job/zowe-install-test/job/add-HLQ-DSN/15/consoleFull
Please test the Pull request before bouncing it back.

Reply:
Messages about ZWESECUR are expected because the ZWESECUR file is only in stagings/cupids pending its merge into staging.  I'll comment this out till then.
I will test the PR and let you know.